pkg/gitprovider/bitbucket.go

```diff
@@ -31,6 +44,18 @@ func NewBitbucketGitProvider(username string, token string) *BitbucketGitProvide
 	return provider
 }
 
+func NewBitbucketServerGitProvider(username string, token string, baseApiUrl *string) *BitbucketServerGitProvider {
```

Comment:
The BitbucketServerGitProvider should go into it's own file.

Reply:
done